test/iterative_verifier_test.go

```diff
@@ -222,6 +297,25 @@ func (t *IterativeVerifierTestSuite) InsertRowInDb(id int, data string, db *sql.
 	t.Require().Nil(err)
 }
 
+func (t *IterativeVerifierTestSuite) InsertCompressedRowInDb(id int, data string, db *sql.DB) {
+	t.SetColumnType(testhelpers.TestSchemaName, testhelpers.TestCompressedTable1Name, testhelpers.TestCompressedColumn1Name, "MEDIUMBLOB", db)
```

Comment:
This seems weird. Why do we change the schema during the test (rather than to just always have it be a MEDIUMBLOB and set it during test setup or whenever we create the db)?

Reply:
We change to `MEDIUMBLOB` because of the snappy-compressed data. `TEXT` expects utf8 data and throws an error when we try to insert the compressed data. I didn't want to change any other tests but the ones added in this PR. If we know later it won't negatively impact the other tests then I'm sure we can change it and just use `MEDIUMBLOB` everywhere.